packages/cli-e2e/utils/cli.ts

```diff
@@ -9,6 +9,16 @@ export function killCliProcess(cliProcess: ChildProcessWithoutNullStreams) {
   return waitForKill;
 }
 
+export function killCliProcessFamily(
+  cliProcessLeader: ChildProcessWithoutNullStreams
+) {
+  const waitForKill = new Promise<void>((resolve) => {
```

Comment:
You prolly could use promisify?

Reply:
I just copied your `killCliProcess` function :P 
But yeah, we can use use `promisify`

Reply:
Ahahah that ricochet xD